[Enhancement] Support Vertex AI, Bedrock Models (#64)
ServiceNow/SyGra: docs/getting_started/model_configuration.md

  parameters:
    temperature: 0.7

gemini_2_5_pro:
  model_type: vertex_ai
  model: gemini-2.5-pro
  parameters:
    max_tokens: 5000
    temperature: 0.5

bedrock_model:
  model_type: bedrock
  model: anthropic.claude-sonnet-4-5-20250929-v1:0
  parameters:
    max_tokens: 5000
    temperature: 0.5

```

> **Important:**
If you set modify_tokenizer: true for a model, you must provide the corresponding chat template in your environment as SYGRA_<MODEL_NAME>_CHAT_TEMPLATE.
Otherwise, exception will be raised during the model initialization.
---

## LiteLLM provider specifics

> This section summarizes provider-specific configuration and capabilities for models implemented under `sygra/core/models/lite_llm`.

Reviewer: do we need to mention this path as it's a public facing document. We can write somthing like: 

```suggestion
This section summarizes provider-specific configuration and capabilities for models implemented with litellm in sygra. 
```
